packages/signature-verifier-api/Cargo.toml

```diff
@@ -6,9 +6,11 @@ license = "MIT OR Apache-2.0"
 edition = { workspace = true }
 
 [dependencies]
+axelar-wasm-std = { workspace = true, features = ["derive"] }
 cosmwasm-schema = { workspace = true }
 cosmwasm-std = { workspace = true }
 error-stack = { workspace = true }
+msgs-derive = { workspace = true }
```

Comment:
in a followup PR, could you actually move this macro into axelar-wasm-std-derive? I don't think we need to have a separate package for this